cmd/service/execute.go

```diff
@@ -11,11 +13,13 @@ import (
 	"github.com/mesg-foundation/core/api/core"
 	"github.com/mesg-foundation/core/cmd/utils"
 	"github.com/mesg-foundation/core/service"
+	"github.com/mesg-foundation/core/utils/xpflag"
 	"github.com/spf13/cobra"
 	survey "gopkg.in/AlecAivazis/survey.v1"
 )
 
 var executions []*core.ResultData
```

Comment:
This is not used anymore.

Reply:
don't get it, what do you mean? `executions` wasn't touched